### DISCUSSION TITLE
mbminer crash and restart

### DISCUSSION THREAD

**Discussion by jeannjie** [NebuTech/NBMiner#618] [General]
miner crashing randomly and starts mining again.. any fixs on this issues,, i hope nebutekch is reading.

 
![nb](https://user-images.githubusercontent.com/93836231/141281000-5ae5a771-4711-4320-b6bb-ee8dfa39d372.JPG)

**Comment by vamperl001** [comment]
really same for me !! 
but on my side ther is not even an error occuring - just opens a new terminal
